libnopegl/src/pipeline_compat.h

```diff
@@ -38,6 +38,7 @@ struct pipeline_compat *ngli_pipeline_compat_create(struct gpu_ctx *gpu_ctx);
 int ngli_pipeline_compat_init(struct pipeline_compat *s, const struct pipeline_compat_params *params);
 int ngli_pipeline_compat_update_vertex_buffer(struct pipeline_compat *s, int32_t index, const struct buffer *buffer);
 int ngli_pipeline_compat_update_uniform(struct pipeline_compat *s, int32_t index, const void *value);
+int ngli_pipeline_compat_update_uniform_count(struct pipeline_compat *s, int32_t index, const void *value, int32_t count);
```

Comment:
`size_t count` ?

Reply:
Oups, old commit, forgot to update this one, fixed.